okhttp/src/main/java/okhttp3/Dispatcher.java

```diff
@@ -106,6 +107,13 @@ public synchronized int getMaxRequestsPerHost() {
     return maxRequestsPerHost;
   }
 
+  /**
+   * Set a callback to be invoked when the number of
```

Comment:
Probably worthwhile to note that this is fired before the response body has been consumed.